pandas/core/frame.py

```diff
@@ -1570,16 +1570,44 @@ def from_csv(cls, path, header=0, sep=',', index_col=0, parse_dates=True,
 
     def to_sparse(self, fill_value=None, kind='block'):
         """
-        Convert to SparseDataFrame
+        Convert to SparseDataFrame.
+
+        Implement the sparse version of the DataFrame meaning that any data
+        matching a specific value it's omitted in the representation.
+        The sparse DataFrame allows for a more efficient storage.
 
         Parameters
         ----------
         fill_value : float, default NaN
+            The specific value that should be omitted in the representation.
         kind : {'block', 'integer'}
+            The kind of the SparseIndex tracking where data is not equal to
+            the fill value:
+
+            - 'block' tracks only the locations and sizes of blocks of data;
+            - 'integer' keeps an array with all the locations of the data.
+
+            The kind 'block' is recommended since it's more memory efficient.
 
         Returns
         -------
         y : SparseDataFrame
+
+        See Also
+        --------
+        DataFrame.to_dense :
+            converts the DataFrame back to the its dense form
```

Comment:
Can you capitalize the first letter and add a period at the end?

I think the description should start in the same line as `to_dense` and continue to the next line indented when it doesn't fit.